src/genie/libs/parser/nxos/tests/test_show_track.py

```diff
@@ -0,0 +1,59 @@
+import re
```

Comment:
if you have folder based testing, no need to have this python unittest style.

Reply:
removed the file